app/templates/_karma.conf.js

```diff
@@ -17,7 +17,10 @@ module.exports = function (config) {
 			'app/app.js',
 			'.generated/templates.js'<% if (useRouting) { %>,
 			'.generated/app/modules/routes/routes.concat.js'<% } %>,
-			'.generated/app/**/*.concat.js' // Leave this last!!!
+			'app/modules/**/*.module.js',
```

Comment:
Let's talk more about it sometime very soon. I'm not totally against it.
Maybe have a hangout sometime tomorrow?
On Mon, Feb 8, 2016 at 8:16 PM John Woodruff <notifications@github.com>
wrote:

> In app/templates/_karma.conf.js
> <https://github.com/erikdonohoo/generator-ng-gulp/pull/4#discussion_r52263288>
> :
>
> > @@ -17,7 +17,10 @@ module.exports = function (config) {
> >  			'app/app.js',
> >  			'.generated/templates.js'<% if (useRouting) { %>,
> >  			'.generated/app/modules/routes/routes.concat.js'<% } %>,
> > -			'.generated/app/**/*.concat.js' // Leave this last!!!
> > +			'app/modules/**/*.module.js',
>
> In all my testing I've never had a single issue with that. But if you feel
> that's important I can try to make the other way work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erikdonohoo/generator-ng-gulp/pull/4/files#r52263288>.
>